thoth/metrics_exporter/jobs.py

```diff
@@ -78,16 +78,21 @@ def count_graph_sync_job_status(job_list_items: list) -> dict:
             graph_sync_jobs_status[job_type]["succeeded"] += 1
         elif "failed" in item["status"].keys():
             graph_sync_jobs_status[job_type]["failed"] += 1
+            if "BackoffLimitExceeded" in item["status"]["conditions"][0]["reason"]:
+                graph_sync_jobs_status[job_type]["retry"] += 1
         elif "active" in item["status"].keys():
             graph_sync_jobs_status[job_type]["active"] += 1
         elif "pending" in item["status"].keys():
             graph_sync_jobs_status[job_type]["pending"] += 1
         elif not item["status"].keys():
-            graph_sync_jobs_status[job_type]["empty"] += 1
-        elif "BackoffLimitExceeded" in item["status"]["conditions"][0]["reason"]:
-            graph_sync_jobs_status[job_type]["retry"] += 1
+            graph_sync_jobs_status[job_type]["waiting"] += 1
         else:
-            _LOGGER.error("Unknown job status %r", item["status"])
+            try:
+                if "BackoffLimitExceeded" in item["status"]["conditions"][0]["reason"]:
+                    graph_sync_jobs_status[job_type]["retry"] += 1
+                    graph_sync_jobs_status[job_type]["failed"] += 1
```

Comment:
Do we want to increment both? I would say `retry` is not relevant here as the job is marked as failed (and we should alert in that case).

Reply:
Initially I thought the same but there are cases where it's "failed" but you don't find the key in the "status", but if you look at ```item["status"]["conditions"][0]``` it states the is failed due to BackoffLimitExceeded

Reply:
OK, we can still alert on `failed` as its basically subset of `retry`. Thanks!

Reply:
Thank you!!